Tests/CarthageKitTests/SimulatorSpec.swift

```diff
@@ -146,6 +146,26 @@ class SimulatorSpec: QuickSpec {
 					}
 				}
 
+				context("Xcode 10.1 after installing 10.2 beta") {
```

Comment:
[IMO] I think this context is not good.

Because this name confuses us as if behaviors change with install states of 10.1.

This context should describe this condition.
Like `with unavailable newer simulators`.

Reply:
I think I addressed this the way you suggested, let me know if I interpreted you incorrectly.

Reply:
OK I think I understand now -- I was getting stuck on the "should" that prefixed the surrounding context labels. Made another change.

Reply:
No. I meant about sentence in `context`.
`Xcode 10.1 after installing 10.2 beta` is not properly named because by described reasons.

```suggestion
				context("when the latest installed simulator is unavailable") {
```

Sorry to confuse you.